main.nf

```diff
@@ -1081,6 +1075,10 @@ process busco_download_db {
 }
 
 metabat_bins
+    .flatMap { it -> def list = []
+                     for (c in it[2]){ list << [ it[0], it[1], c] }
+                     list
+    }
```

Comment:
This seems so much less readable then "mode flatten", are you sure this is the right way?

Reply:
:D no, unfortunately I am not, but I did not find any other way to achieve this.

So we have in channel `metabat_bins`:
[assembler, sample1, [bin1, bin2, bin3]]
[assembler, sample2, [bin7, bin8]]
...

and need:
[assembler, sample1, bin1]
[assembler, sample1, bin2]
[assembler, sample1, bin3]
[assembler, sample2, bin7]
[assembler, sample2, bin8]
...

@ggabernet do you know by any chance a way how this could be achieved differently?

Reply:
I actually also had a look and no better idea. But this replacement for `mode flatten` seems not like an improvement to me. But of course, if this is the only way to replace it, so be it.

Reply:
Just checked out the `transpose` operator, which should be doing exactly this: https://www.nextflow.io/docs/latest/operator.html#transpose

Reply:
This seems definitely nicer. Good catch!